src/bci_build/package/389-ds/README.md.j2

```diff
@@ -0,0 +1,67 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+[389 Directory Server](https://www.port389.org/) is a highly usable, fully featured, reliable and secure LDAP server implementation.
+
+## Usage
+
+By default, the image launches 389 Directory Server with the same configuration that comes with the SUSE Linux Enterprise Server, with one exception, a pre-configured Name Service Switch (NSS) configuration file (`/etc/nsswitch.conf`) in included.
+
+```ShellSession
+$ podman run -it --rm -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Volumes
+
+The database is stored in the directory `/data`. A new empty database will be created on container startup unless an existing database is already present in `/data`.
+
+To mount a host directory as a volume for your database, run the following command:
+
+```ShellSession
+$ podman run -it --rm -v /my/own/datadir:/data:Z -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Certificates
+
+By default, the container will use a self-signed CA certificate and a server certificate signed by that CA.
+
+A custom TLS certificate should be placed in `/data/tls/server.crt` and the key should be placed in and `/data/tls/server.key`.
```

Comment:
We should mention in which format the certificate & key are expected to be (I guess it's PEM)

Reply:
Yes, good point, log messages mention PEM.

Reply:
Yes, it's PEM.

Reply:
added PEM